core/controllers/release_coordinator.py

```diff
@@ -58,6 +59,148 @@ def delete(self) -> None:
         self.render_json({})
 
 
+class UserGroupHandlerNormalizePayloadDict(TypedDict):
+    """Dict representation of UserGroupHandler's normalized_payload
+    dictionary.
+    """
+
+    user_group_name: Optional[str]
+    user_group_user_usernames: Optional[List[str]]
+    user_group_id: Optional[str]
+
+
+class UserGroupHandler(
+    base.BaseHandler[
+        UserGroupHandlerNormalizePayloadDict, Dict[str, str]]
+):
+    """Handler for user groups."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS: Dict[str, str] = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {},
+        'PUT': {
+            'user_group_id': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            },
+            'user_group_name': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            },
+            'user_group_user_usernames': {
+                'schema': {
+                    'type': 'list',
+                    'items': {
+                        'type': 'basestring'
+                    }
+                },
+                'default_value': None
+            }
+        },
+        'DELETE': {
+            'user_group_id': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            }
+        },
+        'POST': {
```

Comment:
Normally we arrange in the order GET/POST/PUT/DELETE, I think. Ditto for the handlers below.

Reply:
Updated